llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -51,195 +51,194 @@ def SplitDoubleTy : DXILOpParamType;
 
 class DXILOpClass;
 
-defset list<DXILOpClass> OpClasses = {
```

Comment:
The formatting changes make this diff a lot harder to read and much larger than it really should be. In general if you have a large format-only diff it is best to put that in a separate NFC PR so that the functional change can be reviewed separately.

Can you break this out into a separate formatting-only NFC change?